service-front/assets/js/lpa/lpa.templates.js

```diff
@@ -41,19 +41,6 @@ this["lpa"]["templates"]["dialog.confirmRepeatApplication"] = Handlebars.templat
     + "</a>\n    </div>\n\n</div>\n";
 },"useData":true});
 
-this["lpa"]["templates"]["errors.formElement"] = Handlebars.template({"compiler":[8,">= 4.3.0"],"main":function(container,depth0,helpers,partials,data) {
```

Comment:
I assume this all redundant? As I thought this was error HTML was complied in php?

Reply:
The same with the HB templates as well I assume?

Reply:
Yes, I removed the HTML template as it's not used anywhere.